sigs.yaml

```diff
@@ -678,20 +678,10 @@ sigs:
       slack: sig-cluster-lifecycle
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle
       teams:
-      - name: sig-cluster-lifecycle-api-reviews
-        description: API Changes and Reviews
-      - name: sig-cluster-lifecycle-bugs
-        description: Bug Triage and Troubleshooting
-      - name: sig-cluster-lifecycle-feature-requests
-        description: Feature Requests
-      - name: sig-cluster-lifecycle-misc
-        description: General Discussion
+      - name: sig-cluster-lifecycle
+        description: Notify group
```

Comment:
Are we supposed to keep this one around? Or just nuke all of them?

Reply:
Yes we need at least one.